utils/bazel/llvm-project-overlay/mlir/BUILD.bazel

```diff
@@ -1231,6 +1231,15 @@ td_library(
     deps = [":OpBaseTdFiles"],
 )
 
+td_library(
```

Comment:
why is this so far away from Sparse?

I think I would prefer to just put this into the "SparseTensorTdFiles" rule, and depend on that (even though that pulls in more TD files, but that is what is generally done)